src/components/pages/summary/SummaryPage.js

```diff
@@ -0,0 +1,127 @@
+import propTypes from 'prop-types'
+import React from 'react'
+import {
+  getLowerThirdTimeCodeIn,
+  getLowerThirdTimeCodeOut,
+  getTimecodeTotalLength,
+  getTimecodeTotalLengthFromSequenceCards,
+} from '../../../utils/Timecode'
+import EventCard from '../../EventCard/EventCard'
+
+SummaryPage.propTypes = {
+  sequenceCards: propTypes.object.isRequired,
+}
+
+export default function SummaryPage({ sequenceCards }) {
```

Comment:
I would outsource your first and second half components. You have to scroll down really far on this page to get to your SummaryPage.

Reply:
Yeah, i will refactore the shit out of it. This version is only to get it work. ;-)